stdlib/mimalloc_jll/test/runtests.jl

```diff
@@ -0,0 +1,9 @@
+# This file is a part of Julia. License is MIT: https://julialang.org/license
+
+using Test, Libdl, mimalloc_jll
```

Comment:
`Libdl` is unused?

Reply:
I copied the libuv one, but I think it's wrong in both of them :)